imports/blocks/give/fieldsets/confirm/ScheduleLayout.js

```diff
@@ -7,10 +8,15 @@ import {
   ScheduleItem,
 } from "../shared";
 
+type IHeader = {
+  override?: Object,
```

Comment:
Wouldn't this be a Function? Or any React class / function? Is there a ReactElement type we can use?